portal-frontend/src/app/features/applications/edit-submission/parcel-details/parcel-entry/parcel-entry.component.ts

```diff
@@ -412,13 +406,15 @@ export class ParcelEntryComponent implements OnInit {
     });
   }
 
-  onOwnerEdited() {
+  onOwnerEdited(updatedOwner: any) {
```

Comment:
Instead of any can you use ApplicationOwnerDto ?

Reply:
Yes, definitely. I had meant to use any as a placeholder just to see if this would work.